esy-package-config/Dist.re

```diff
@@ -112,8 +112,9 @@ module Parse = {
 
   let gitOrGithubManifest =
     switch%bind (peek_char) {
-    | Some(':') =>
-      let%bind () = advance(1);
+    | Some(':')
+    | Some('%') =>
+      let%bind () = ignore(string("%3A")) <|> advance(1);
```

Comment:
@melwyn95 Do you think we can warn the user here? That `%3A` is valid URI, but an invalid spec to specify package source. I'm not sure why anyone would necessarily have to use encoded URI components. 

It would nice to ask users to fix them. (Not absolutely necessary for this PR)

Reply:
Merging this PR, but the discussion can continue